geonode_mapstore_client/client/js/plugins/ResourcesGrid.jsx

```diff
@@ -458,11 +463,11 @@ function ResourcesGrid({
     facets
 }, context) {
 
-    const [cardLayoutStyle, setCardLayoutStyle] = useLocalStorage('layoutCardsStyle', 'grid');
+    const [cardLayoutStyleState, setCardLayoutStyle] = useLocalStorage('layoutCardsStyle', cardLayoutStyle || defaultLayoutCardsStyle);
```

Comment:
I think we should avoid to use the localStorage info if cardLayoutStyle is defined, something like this:
```js
const [_cardLayoutStyleState, setCardLayoutStyle] = useLocalStorage('layoutCardsStyle', defaultLayoutCardsStyle);
const cardLayoutStyleState = cardLayoutStyle || _cardLayoutStyleState;
```

in this way if other pages has the resource grid with the possibility to select that will not have effect on this specific configuration